src/uint256.h

```diff
@@ -89,6 +89,14 @@ class base_blob
                ((uint64_t)ptr[7]) << 56;
     }
 
+    std::array<uint8_t, WIDTH> GetRawBytes() const
+    {
+        std::array<uint8_t, WIDTH> buf = {};
+        memcpy(buf.data(), this->begin(), WIDTH);
```

Comment:
```suggestion
        std::memcpy(buf.data(), this->begin(), WIDTH);
```
It probably works without the `std::` (or `use std;`) by coincidence of the included headers implicitly declaring `memcpy` in the global namespace. Strictly speaking that's not portable (similarly for the uses of `memset` in this file).